src/WebApp/Pages/Account/Login.cshtml

```diff
@@ -2,18 +2,34 @@
 @model LoginModel
 
 <form asp-page="/account/login" method="post">
+    <div class="has-padding-2">
+        <h1>Sign in</h1>
+        <p>Welcome to Codidact! You can login to your account here. Don't have an account? <a asp-page="Register">Sign up instead.</a></p>
+    </div>
     <input asp-for="ReturnUrl" type="hidden" />
 
-    <label asp-for="Email">Email:</label>
-    <input asp-for="Email" /><br>
+    <div class="widget">
+        <div class="widget--body">
+            <div class="has-padding-1">
+                <label asp-for="Email" class="form-element">E-Mail Address</label>
+                <input asp-for="Email" type="email" class="form-element" id="email">
+            </div>
 
-    <label asp-for="Password">Password:</label>
-    <input asp-for="Password" /><br>
 
-    <label>Remember Login</label>
-    <input asp-for="RememberLogin" /><br>
+            <div class="has-padding-1">
+                <label asp-for="Password" class="form-element">Password</label>
+                <input asp-for="Password" class="form-element" type="password">
+            </div>
 
-    <button>Login</button>
+            <div class="has-padding-1">
+                <label asp-for="RememberLogin" class="form-element">Remember Login</label>
+                <input asp-for="RememberLogin" class="form-checkbox-element" type="checkbox">
+            </div>
+        </div>
+        <div class="widget--footer">
+            <button type="submit" class="button is-filled">Log in</button>
+        </div>
+    </div>
 
     <partial name="_ValidationSummary" />
```

Comment:
Applies to register, too, but where is this partial defined? How is it styled? Shouldn't the validation errors be shown above the `.widget` inside a `.notice.is-danger`?

Reply:
Can you give an example to this design?

Reply:
This line should be moved right before the widget

[This file](https://github.com/misha130/authentication/blob/register-page/src/WebApp/Pages/Shared/_ValidationSummary.cshtml) should be updated to something like this:

```
@if (!ViewContext.ModelState.IsValid) {
    <div class="notice is-danger">
        <div asp-validation-summary="All"></div>
    </div>
}
```

Reply:
So I did what you said:
![image](https://user-images.githubusercontent.com/6649900/77788852-5d0f8880-7038-11ea-921a-ae211f777ed9.png)

The only issue is the validation messages:

- Password and Password Confirmation must Match
- E-Mail Address is Required
- Display Name is Required
- Password is Required
- Password Confirmaton is Required

I am not sure about them

Reply:
They should be all lowercase, except for the first character of the first word IMHO.